rust/token-metadata/program/src/deprecated_instruction.rs

```diff
@@ -1,14 +1,12 @@
-use {
-    crate::{
-        instruction::{CreateMasterEditionArgs, MetadataInstruction},
-        state::Reservation,
-    },
-    borsh::{BorshDeserialize, BorshSerialize},
-    solana_program::{
-        instruction::{AccountMeta, Instruction},
-        pubkey::Pubkey,
-        sysvar,
-    },
+use crate::{
```

Comment:
I'm realizing that here we end up with multiple `use` statements (this is how it was in tests, just less organized).
We should agree on one approach and if it is different than the code is at this point I can adapt the `rustfmt.toml` and this PR to respect it.